perl/bin/bystro-server.pl

```diff
@@ -279,7 +279,8 @@ sub coerceInputs {
     compress       => 1,
     archive        => 1,
```

Comment:
I think you forgot to get rid of this archive here since you specify archive => 0 in your new code on line 283. 

Also, on line 267 commonArgs has an extra m but if that's intentional since it seems to be like that in other spots too just me know.

Reply:
oh good catch!